src/PoolManager.sol

```diff
@@ -87,13 +87,13 @@ contract PoolManager is IPoolManager, ProtocolFees, NoDelegateCall, ERC6909Claim
         return pools[id].positions.get(_owner, tickLower, tickUpper);
     }
 
-    function isLockSet() external view returns (bool) {
-        return Lock.isLocked();
+    function isUnlocked() external view override returns (bool) {
+        return Lock.isUnlocked();
     }
 
-    /// @notice This will revert if the contract is not locked
-    modifier isLocked() {
-        if (!Lock.isLocked()) revert ManagerNotLocked();
+    /// @notice This will revert if the contract is locked
+    modifier onlyWhenUnlocked() {
+        if (!Lock.isUnlocked()) revert ManagerNotUnlocked();
```

Comment:
hmmm maybe good to avoid double negatives .. would at the very least change the error to be 
ManagerLocked().